cpp/ql/lib/semmle/code/cpp/dataflow/ProductFlow.qll

```diff
@@ -1,10 +1,29 @@
-import semmle.code.cpp.ir.dataflow.DataFlow
+/**
```

Comment:
How about putting it in `dataflow/internal`, for now? Eventually, this probably be in the shared `dataflow` pack?

Reply:
We could definitely do that, yeah 👍. I guess that means I should also revert the change note commit since we don't want users importing internal libraries.

Reply:
I've force-pushed a new commit that moves the library to `dataflow/internal` now. I've also reverted the change note since we don't want people to start using the library yet now that it's in an internal folder.